math/scalapack/Portfile

```diff
@@ -2,16 +2,15 @@
 
 PortSystem          1.0
 PortGroup           cmake 1.1
-PortGroup           mpi 1.0
+PortGroup           compiler_blacklist_versions 1.0
 PortGroup           linear_algebra 1.0
+PortGroup           github 1.0
+PortGroup           mpi 1.0
 PortGroup           muniversal 1.0
-PortGroup           compiler_blacklist_versions 1.0
 
-name                scalapack
-version             2.2.0
-revision            2
+github.setup        Reference-ScaLAPACK scalapack 2.2.1 v
+revision            0
 categories          math
-platforms           darwin
 maintainers         {@catap korins.ky:kirill} openmaintainer
```

Comment:
please rebase with `master`; I am not sure if we otherwise will re-add him as maintainer

Reply:
@reneeotten Done